### DISCUSSION TITLE
Request to add option of multiple GitHub account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#8069] [Mobile]
Please add an option for changing between accounts for the GitHub android app.Everytime logging in and logging out is difficult. I use one account as my official account and other as experimental in which I write code and do experiments and when they work well I put them on my official account. On clicking the plus button in the account section of the app it does not show add personal account option, shows about add company.Please add the option of adding personal account also.
Thank You 😊

**Comment by brianlovin** [comment]
Thanks for the suggestion @Dave596 — supporting multiple personal accounts is not on our roadmap right now, so won't be available in the near term, but I've added your +1 to our tracking issues so the team can take a look and prioritize this work in the future.

**Reply by MohakDave** [reply]
Thank you
